addons/mail/static/src/core_ui/message.xml

```diff
@@ -141,6 +114,33 @@
                                     </Dropdown>
                                 </div>
                             </div>
+                                <t t-if="message.type !== 'notification' and !message.isTransient and (message.hasTextContent or message.subtypeDescription)">
+                                    <LinkPreviewList t-if="!state.isEditing and message.linkPreviewSquash" linkPreviews="message.linkPreviews" deletable="false"/>
+                                    <t t-else="">
+                                        <div class="position-relative">
+                                            <div class="o-mail-Message-bubble rounded-bottom-3 position-absolute top-0 start-0 w-100 h-100" t-att-class="{
```

Comment:
Should indent `<t>` with `</div>`: should start on same indent level
Make sure to properly indent content in negative diff